app/jobs/get_suggestions_job.rb

```diff
@@ -0,0 +1,93 @@
+class GetSuggestionsJob < ApplicationJob
+  queue_as :default
+
+  def perform(job_status)
+    @job_id = job_status.job_id
+
+    send_spotify_request
+
+    if get_spotify_tracks
```

Comment:
It's not clear to me what this means. Does `get_spotify_tracks` return `true`? Does it set data somewhere?

If it returns data, does it make sense to get that data in this method to be passed along to whatever you are updating?